### ISSUE TITLE
Dynamic Import Statements not parsed correctly

### ISSUE THREAD

**Issue by Johnfred24** [trullock/NUglify#163]
I have following Statement in my Code:
```
import("/js/swiper.js").then(_ => {
})
```

this is changed to the following code by the minifier
```
import;"/js/swiper.js".then(()=>{})
```

I also receive an Error in the Result:
>  Import statement must include module name, Code JS1328

**Comment by trullock** [comment]
Curious, seems the parser is being a bit naive with `import` statements. Where does your import function come from btw?

**Comment by trullock** [comment]
This should be fixed in 1.9.8
